internal/ssm/install.go

```diff
@@ -19,13 +21,46 @@ import (
 )
 
 const (
-	installerPath = "/opt/ssm/ssm-setup-cli"
-	configRoot    = "/etc/amazon"
+	installerPath   = "/opt/ssm/ssm-setup-cli"
```

Comment:
this key is in the progress of being rotated, it expires next week. Ill wait to merge this PR until we get a new release of ssm using the new key for the signature.

Reply:
I was also thinking it may make sense to add this to our nodeadm manifest instead of built into the code. Open to thoughts here?

Reply:
i vote to have it here for now
i like the idea of having it in the manifest but, that doesn't really work well (from a security pov) until we sign the manifest itself.

Reply:
yeah we can add it the manifest once we get signing in.